### DISCUSSION TITLE
deta.sh can't run python script

### DISCUSSION THREAD

**Discussion by x3221** [deta/cloud-docs#661] [Help]
The script works fine locally but when I deploy it 'detalib' gets added to my requirements.txt and then I get an error which says `Unable to import module '_entry': No module named 'detalib'`.

Does anyone know how to fix this?

**Comment by jnsougata** [comment]
can you try adding `deta` to `requirements.txt`

**Reply by x3221** [reply]
> can you try adding `deta` to `requirements.txt`

Deta was in the requirements.txt and everything ran fine locally. I circumvented the issue by using the deta api instead of the python module.
